lib/models/command.js

```diff
@@ -178,6 +178,38 @@ var Command = CoreObject.extend({
   },
 
   /**
+   * Called when command is interrupted from outside, e.g. ctrl+C or process kill
+   * Can be used to cleanup artifacts produced by command and control process exit code
+   *
+   * @method onInterrupt
+   * @return {Promise|undefined} if rejected promise then result of promise will be used as an exit code
+   */
+  onInterrupt: function() {
+    if (this._currentTask) {
+      return this._currentTask.onInterrupt();
+    }
```

Comment:
Agree, I think the same question is relevant for the case when the `cli/cli` is interrupted when there is still no active command.